src/main/java/com/google/api/codegen/discovery/viewmodel/SampleView.java

```diff
@@ -41,6 +41,8 @@ public String resourceRoot() {
 
   public abstract String apiVersion();
 
+  public abstract String appName();
```

Comment:
If you set `@Nullable` you don't have to provide it in any other language transformer.